docs/design/versioning.md

```diff
@@ -34,19 +34,36 @@ Documentation for other releases can be found at
 * **API vX[betaY]** refers to the version of the HTTP API.
 
 ## Release versioning
 
 ### Minor version scheme and timeline
 
-* Kube X.Y.0-alpha.W, W > 0: Alpha releases are released roughly every two weeks directly from the master branch. No cherrypick releases. If there is a critical bugfix, a new release from master can be created ahead of schedule.
-* Kube X.Y.Z-beta.W: When master is feature-complete for Kube X.Y, we will cut the release-X.Y branch 2 weeks prior to the desired X.Y.0 date and cherrypick only PRs essential to X.Y.  This cut will be marked as X.Y.0-beta.0, and master will be revved to X.Y+1.0-alpha.0.  If we're not satisfied with X.Y.0-beta.0, we'll release other beta releases, (X.Y.0-beta.W | W > 0) as necessary.
-* Kube X.Y.0: Final release, cut from the release-X.Y branch cut two weeks prior.  X.Y.1-beta.0 will be tagged at the same commit on the same branch.  X.Y.0 occur 3 to 4 months after X.Y-1.0.
-* Kube X.Y.Z, Z > 0: [Patch releases](#patch-releases) are released as we cherrypick commits into the release-X.Y branch, (which is at X.Y.Z-beta.W,) as needed.  X.Y.Z is cut straight from the release-X.Y branch, and X.Y.Z+1-beta.0 is tagged on the same commit.
+* Kube X.Y.0-alpha.W, W > 0 (Branch: master)
+  * Alpha releases are released roughly every two weeks directly from the master branch.
+  * No cherrypick releases. If there is a critical bugfix, a new release from master can be created ahead of schedule.
+* Kube X.Y.Z-beta.W (Branch: release-X.Y)
+  * When master is feature-complete for Kube X.Y, we will cut the release-X.Y branch 2 weeks prior to the desired X.Y.0 date and cherrypick only PRs essential to X.Y.
+  * This cut will be marked as X.Y.0-beta.0, and master will be revved to X.Y+1.0-alpha.0.
+  * If we're not satisfied with X.Y.0-beta.0, we'll release other beta releases, (X.Y.0-beta.W | W > 0) as necessary.
+* Kube X.Y.0 (Branch: release-X.Y)
+  * Final release, cut from the release-X.Y branch cut two weeks prior.
+  * X.Y.1-beta.0 will be tagged at the same commit on the same branch.
+  * X.Y.0 occur 3 to 4 months after X.Y-1.0.
+* Kube X.Y.Z, Z > 0 (Branch: release-X.Y)
+  * Patch releases are released as we cherrypick commits into the release-X.Y branch, (which is at X.Y.Z-beta.W,) as needed.
+  * X.Y.Z is cut straight from the release-X.Y branch, and X.Y.Z+1-beta.0 is tagged on the same commit.
+* Kube X.Y.Z-patch.W, Z > 0 (Branch: release-X.Y.Z)
+  * Cut release-X.Y.Z branch to hold the isolated patch release
+  * Tag release-X.Y.Z branch + fixes with X.Y.Z-patch.W
+  * [Branched patch releases](#branched-patch-releases) are rarely needed but used for emergency/critical fixes to the latest release
+  * These are special and different in that the latest tag is branched to isolate the emergency/critical fix from all other changes that have landed since the previous tag
```

Comment:
But, maybe move this explanation to the top, before the details about how it's cut?